.editorconfig

```diff
@@ -52,7 +52,7 @@ dotnet_style_explicit_tuple_names = true:error
 # Prefer "var" everywhere
 csharp_style_var_for_built_in_types = true:error
 csharp_style_var_when_type_is_apparent = true:error
-csharp_style_var_elsewhere = true:error
+csharp_style_var_elsewhere = false

```

Comment:
I think this should be changed to `true` without `:error`. `var` should be preferred everywhere, until the developer makes the decision to put the type in because of reasons. Changing this to `false` will incorrectly instruct VS to put the concrete type everywhere where it isn't a built in type or where it is apparent.

Reply:
Ref: https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0007-ide0008